src/coordinator/storage/local/storage.go

```diff
@@ -161,11 +264,46 @@ func (s *localStorage) FetchBlocks(
 	return res, nil
 }
 
+func (s *localStorage) Close() error {
+	return nil
```

Comment:
Add a //TODO so we don't forget to close later?

Reply:
Hm, this was here before I believe?